toolkit/scripts/pkggen.mk

```
@@ -43,10 +43,15 @@ logging_command = --log-file=$(LOGS_DIR)/pkggen/workplan/$(notdir $@).log --log-
 $(call create_folder,$(LOGS_DIR)/pkggen/workplan)
 $(call create_folder,$(rpmbuilding_logs_dir))
 
-.PHONY: clean-workplan clean-cache clean-cache-worker clean-grapher-cache-worker clean-spec-parse clean-ccache graph-cache analyze-built-graph workplan
+.PHONY: clean-workplan clean-cache clean-cache-worker clean-grapher-cache-worker clean-spec-parse clean-ccache graph graph-cache graph-preprocessed analyze-built-graph workplan
+##help:target:parsed-specs=Parse package specs and generate a specs.json file encoding all dependency information.
+parsed-specs: $(specs_file)
```

nit: do we want to rename this to parse-specs, as other targets sound like 'this target _wil_ ..' and this one sounds like 'this target _has_ ..'.

---

Sure, that seems reasonable.
```suggestion
parse-specs: $(specs_file)
```